src/main/markdown/gettingstarted-v2.md

```diff
@@ -23,7 +23,7 @@ the plugin generates a Maven project with separate modules for
 
 ## Prerequisites<a id="prereqs"></a>
 
-1.  GWT 2.11.0 still supports Java 8 (running and building). We suggest to use at least Java 11 or 17. If necessary, download and
+1.  GWT 2.12.0 still supports Java 8 (running and building). We suggest to use at least Java 11 or 17. If necessary, download and
```

Comment:
I thought Java 8 support dropped with #9899 or I misunderstood?

Reply:
AFAIK, you still need at least Java 11 to run GWT, but your project is able to build and run for Java 8. We had that questions a few days ago. See: https://github.com/gwtproject/gwt/pull/10019#pullrequestreview-2397521908

Reply:
Thanks for clarifying  👍🏻